### PR TITLE
Change footer message

### PR DESCRIPTION
**Pull request by Jake-Coding**
use discord username conventions (username#discriminator instead of username:discriminator)

### CODE REVIEW

File: MomentumDiscordBot/Commands/General/GeneralModule.cs [hunk 1]
```diff
@@ -159,4 +159,4 @@ public class GeneralModule : MomentumModuleBase
             await context.CreateResponseAsync(embed: embedBuilder.Build(), true);
         }
     }
-}
```
Comment (tsa96) [inline MomentumDiscordBot/Commands/General/GeneralModule.cs]: ?? Did you accidentally add a windows line ending or something? I'd prefer small fix PRs like this include absolutely no changes than the line they're modifying. Also when removing this make sure to not create further commits, easiest to use `git reset --soft HEAD~1`, discard the this formatting thing, recommit, then `git push --force`.

**Review by tsa96 — CHANGES_REQUESTED** [review]

File: MomentumDiscordBot/Commands/General/GeneralModule.cs (hunk 1 above)
Comment (Jake-Coding) [reply MomentumDiscordBot/Commands/General/GeneralModule.cs]: oop yeah id assume it added some dumb line ending

Comment (Jake-Coding) [reply MomentumDiscordBot/Commands/General/GeneralModule.cs]: should be fixed now

**Review by tsa96 — APPROVED** [review]
Great, thank you!!

**Pull request merged**